addons/test_mail/tests/test_performance.py

```diff
@@ -501,7 +501,7 @@ def test_mail_composer_w_template_form_attachments(self):
             )
             composer = composer_form.save()
 
-        with self.assertQueryCount(__system__=53, employee=70):
+        with self.assertQueryCount(__system__=54, employee=73):
```

Comment:
This change comes with the addition of the "thumbnail" field on web_editor's ir_attachment. The two solutions that fixed this test were adding `prefetch=True` to thumbnail, which IMO was a bad idea, and bumping up the query count on this test specifically. If there is any other solution that comes to mind instead of changing the test, I am open to it 🤔